packages/protocol/package.json

```diff
@@ -22,7 +21,7 @@
     "test:integration": "TEST_TYPE=integration ./test/test_integration.sh",
     "test:tokenomics": "TEST_TYPE=tokenomics ./test/test_integration.sh",
     "test:all": "pnpm run test && pnpm run test:integration && pnpm run test:tokenomics && pnpm run test:genesis",
-    "deploy:hardhat": "./scripts/download_solc.sh && LOG_LEVEL=debug pnpm hardhat deploy_L1 --network hardhat --dao-vault 0xdf08f82de32b8d460adbe8d72043e3a7e25a3b39 --team-vault 0xdf08f82de32b8d460adbe8d72043e3a7e25a3b39 --l2-genesis-block-hash 0xee1950562d42f0da28bd4550d88886bc90894c77c9c9eaefef775d4c8223f259 --bridge-funder-private-key ddbf12f72c946bb1e6de5eaf580c51db51828ba198d9b0dba9c7d48ec748dc04 --bridge-fund 0xff --oracle-prover 0xdf08f82de32b8d460adbe8d72043e3a7e25a3b39 --solo-proposer 0xdf08f82de32b8d460adbe8d72043e3a7e25a3b39 --confirmations 1",
+    "deploy:foundry": "./script/download_solc.sh && PRIVATE_KEY=ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 ORACLE_PROVER_ADDRESS=0xa0Ee7A142d267C1f36714E4a8F75612F20a79720 SOLO_PROPOSER=0xa0Ee7A142d267C1f36714E4a8F75612F20a79720 PROXY_ADMIN=0x70997970C51812dc3A010C7d01b50e0d17dc79C8 TAIKO_L2_ADDRESS=0xa0Ee7A142d267C1f36714E4a8F75612F20a79720 TAIKO_TOKEN_PREMINT_RECIPIENT=0xa0Ee7A142d267C1f36714E4a8F75612F20a79720 TAIKO_TOKEN_PREMINT_AMOUNT=0xffff L2_GENESIS_HASH=0xee1950562d42f0da28bd4550d88886bc90894c77c9c9eaefef775d4c8223f259 L2_CHAIN_ID=167001 forge script script/deploy.s.sol:DeployOnL1 --fork-url http://localhost:8545 --broadcast --ffi -vvvv --via-ir",
```

Comment:
just a comment, not a feedback: FYI, you can start an anvil node with a mnemonic as well as a private key, could be useful for testing using multiple accounts.